### DISCUSSION TITLE
Not echo an error/warning?

### DISCUSSION THREAD

**Discussion by MatejaSrejic** [PHPMailer/PHPMailer#2229] [Q&A] [answered]
So, I have the same exact code as the example in the README, but for some reason, it **always** echos the error even though I already have the catch statement.

**Comment by Synchro** [comment]
The example in the readme has debug output enabled, so you will always see that, and it echos error messages when it catches an exception, so it sounds like it’s behaving exactly as it should. I don’t see a problem here.

**Reply by MatejaSrejic** [reply]
I removed the SMTPDebug line and it is still echo-ing logs. Do I have to set the value to a precise number?

**Comment by MatejaSrejic** [comment, marked as answer]
Found the solution by searching the source - set SMTDebug to 0.
